packages/qwik-city/package.json

```diff
@@ -9,6 +9,7 @@
     "source-map": "^0.7.4",
     "svgo": "^3.3",
     "undici": "*",
+    "valibot": "0.37.0",
```

Comment:
I used a fixed version number because the same approach was taken with Zod. As long as there are no breaking changes, it doesn't matter. So we can change it to `"^0.33.0"` or `">=0.33.0 <1"` and update to v1 when it is available. Please let me know how to proceed.

Reply:
With zod it actually needs the fixed version because the minor update breaks the types :( With valibot I hope that won't happen :)

Reply:
I don't expect breaking changes. Should I change it to `>=0.33.0 <2`?

Reply:
Yes please

Reply:
@fabian-hiller @wmertens apparently there were some changes done for 1.0.0-alpha that caused one of my projects to fail using `globalAction$` because it was upgraded from an older qwik-city version but kept using the older version because it was in the range.

I think we need to get to the standard-schema solution asap to make zod / valibot optional